### DISCUSSION TITLE
Week 2 activities and Worksheet 2

### DISCUSSION THREAD

**Discussion by ebajcar** [ebajcar/web10199_worksheets#9] [Weekly activities]
Post questions, answer others' questions, share observations and code snippets.

**Comment by E-Annie** [comment]
Hello Professor Ellen, 

I was trying to access the class examples listed in worksheet 2 (eg. Fibonacci - console) and all the links state that file is not found. I am unsure if this is a problem only on my end or if it is the url that is not working. 

Thank you! 
E Hyun Annie Kim

**Reply by ebajcar** [reply]
I've updated the links, they should now work. Let me know if not.
You can also find these files [here ](https://github.com/ebajcar/web10199_worksheets/tree/master/syst10199/set2)
https://github.com/ebajcar/web10199_worksheets/tree/master/syst10199/set2
